neonvm/controllers/virtualmachine_controller.go

```diff
@@ -319,6 +324,16 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 		meta.SetStatusCondition(&virtualmachine.Status.Conditions, metav1.Condition{Type: typeAvailableVirtualMachine, Status: metav1.ConditionUnknown, Reason: "Reconciling", Message: "Starting reconciliation"})
 	}
 
+	enableSSH := false
+	if virtualmachine.Spec.EnableSSH != nil && *virtualmachine.Spec.EnableSSH {
+		enableSSH = true
+	}
```

Comment:
IIUC, if we set the default via kubebuilder, we should be ok to assume that it's always set here.

Might also be reasonable to have a `sshEnabled(*vmv1.VirtualMachine)` sort of function to handle this - then when we deprecate `EnableSSH` to make it always-enabled, we're just changing one function.

Reply:
changed it here: b8de84026b60a658e05673f703946682e5580c57